jellyfin_kodi/helper/xmls.py

```diff
@@ -71,3 +71,64 @@ def advanced_settings():
             xbmc.executebuiltin('RestartApp')
 
             return True
+
+def verify_kodi_defaults():
+    ''' Make sure we have the kodi default folder in place.
+    '''
+
+    source_base_path = translate_path("special://xbmc/system/library/video")
+    dest_base_path = translate_path("special://profile/library/video")
+
+    if not os.path.exists(source_base_path):
+        LOG.error("XMLs source path `%s` not found.", source_base_path)
+        return False
```

Comment:
Given that the rest of the function is implicit returns with no type, unless you're planning on putting a check where ever this is called it should probably just be

```suggestion
        return
```
so we're not mixing datatypes.

Reply:
The intent was to add tests, but I won't be doing that now, as I'd have to figure out how to mock the xbmc libraries first.